test/test.js

```diff
@@ -1545,6 +1545,22 @@ exports['wrap EventEmitter (or jQuery) on handler with args wrapping by array']
     });
 };
 
+exports['takeUntil'] = function (test) {
+    a = _();
+    b = _();
+    a.resume();
```

Comment:
Don't call `resume` on either stream. You shouldn't need to. Just move the `c.toArray` above the `write` calls.